notebooks/cleaning_data.ipynb

```diff
@@ -0,0 +1,2892 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    one_hot_features, numeric_features, labeled_features = CLD.get_features_based_type(data, features_st60_missing)
</pre><p>Why does the output display a sample dataframe with the title "Samples of numeric features" if the numeric features list has a length of 0, which indicates that no features are numeric features? It has the same presentation as the "Samples of labeled features" output. please check the function returns.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/mawada-sweis/Clustering-Analysis/pull/21/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='22'/>